src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/XASdkTests.cs

```diff
@@ -606,7 +607,7 @@ void CreateEmptyFile (params string [] paths)
 		public void XamarinLegacySdk ()
 		{
 			var proj = new XASdkProject (outputType: "Library") {
-				Sdk = "Xamarin.Legacy.Sdk/0.1.0-alpha2",
+				Sdk = "Xamarin.Legacy.Sdk/0.1.0-alpha3",
```

Comment:
I had to make Xamarin.Legacy.Sdk support C# 10, so I shipped a release with this fix.